code/game/gamemodes/malfunction/malfunction.dm

```diff
@@ -257,7 +257,7 @@
 		world << "<B>The self-destruction of [station_name()] killed everyone on board!</B>"
 
 	else if ( station_captured &&  malf_dead && !station_was_nuked)
-		feedback_set_details("round_end_result","halfwin - AI killed, staff lost control")
+		feedback_set_details("round_end_result","draw - AI killed, staff lost control")
 		world << "<FONT size = 3><B>Neutral Victory</B></FONT>"

```

Comment:
Way I interpreted it was that since the AI seized control but then died that essentially no one has control anymore, and the station is screwed even though the AI was killed.